include/rocksdb/utilities/write_batch_with_index.h

```diff
@@ -92,37 +92,38 @@ class WriteBatchWithIndex : public WriteBatchBase {
   //                show two entries with the same key.
   explicit WriteBatchWithIndex(
       const Comparator* backup_index_comparator = BytewiseComparator(),
-      size_t reserved_bytes = 0, bool overwrite_key = false);
+      size_t reserved_bytes = 0, size_t max_bytes = 0,
```

Comment:
Not sure if this is okay. I feel like some applications may break in subtle ways. Maybe I should have just added max_bytes to the end?